### PR TITLE
add docker images to README

### PR DESCRIPTION
**Pull request by XinkaiWang515**
Add GCP public docker image address to xgboost-operator README.

### CODE REVIEW

**Comment by merlintang** [comment]
please update the PR title

**Comment by merlintang** [comment]
you need to close the previous one, then send a new one

File: README.md [hunk 1]
```diff
@@ -193,3 +193,14 @@ Events:
   Normal  ExitedWithCode           47s                xgboostjob-operator  Pod: default.xgboost-dist-iris-test-master-0 exited with code 0
   Normal  XGBoostJobSucceeded      47s                xgboostjob-operator  XGBoostJob xgboost-dist-iris-test is successfully completed.
  ```
+
+
+# Docker Images
+
+You can use the following Dockerfiles to build the images yourself:
```
Comment (terrytangyuan) [inline README.md:200]: Dockerfiles -> Dockerfile

File: README.md [hunk 2]
```diff
@@ -193,3 +193,14 @@ Events:
...
+
+* [xgboost-operator](https://github.com/kubeflow/xgboost-operator/blob/master/Dockerfile)
+
+For your convenience, you can download built xgboost-operator image from:
```
Comment (terrytangyuan) [inline README.md:204]: download built -> pull the existing image from the following

**Comment by terrytangyuan** [comment]
Next time you can just update the existing PR. No need to create new PRs.

**Comment by XinkaiWang515** [comment]
Sure, I see. Thank you so much for your advise.

**Pull request merged**